### DISCUSSION TITLE
Unresponsiveness

### DISCUSSION THREAD

**Discussion by ghost** [SpotX-Official/SpotX#322] [Errors]
Why hello there!

For the short two months that I have had this program, I have absolutely been in love! In itself, I'd argue this is nitpicky, but I have noticed that this mishap is becoming seemingly more frequent with time: I boot up my SpotX Spotify application, and I begin to play whatever music from whatever playlist, album, etc. It works fine for a good 30 minutes to an hour, but somewhere near that time frame, it quits responding to my inputs on the pause, skip, rewind, replay, etc. buttons. Additionally, when I do close the application and try to relaunch it, I repeatedly get a "spotify.exe has stopped responding" pop-up message. I have to force-shut it in Task Manager for me to continue on--which I doubt is healthy--and then the cycle repeats itself. I'm sure that this isn't a default Spotify issue since when I had it, it would never behave this way. However, I'm pretty unknowledgeable on program issues or bugs like this, so I figure I should at least mention it to people who know what they're doing--i.e., you! And alas, here I am!

This is more of a minor inconvenience than an excruciating hassle, so if busy, please do not feel obligated to prioritize this issue.

Thank you so!

**Comment by amd64fox** [comment]
[Create issue](https://github.com/amd64fox/SpotX/issues/new?assignees=&labels=%E2%9D%8C+bug&template=bug_report.yml)

**Reply by ghost** [reply]
Hello!

I sincerely apologize for not getting back to you sooner! I've created an issue, but it seems to not have been posted on my end. Perhaps I inserted it into the wrong channel or modification, or something along those lines, but unfortunately, Github nor technology are my fortes 😂 I was curious if you were able to see the issue I created and if there was any development on it. It has gotten a tad more persistent. Additionally, I must ask: are you a SpotX developer? I cannot seem to find a way to locate or message them.

Thank you for all your help, and apologies in advance for the technological challenges!

**Reply by amd64fox** [reply]
your issue has been closed, the [reason](https://github.com/amd64fox/SpotX/issues/325#issuecomment-1438612097) is listed there, if something is not clear to you, then let me know

**Reply by ghost** [reply]
Oh, apologies! I'm not sure what you mean by the a not being filled out and me selecting a checkmark... if there is a form I'm supposed to fill out or a checkmark I should choose, do you mind directing me on how to get there? And I would like to help whoever's necessary to fix this problem, but I can only offer weak assistance with the coding or whathaveyou. That's partially why I reported the issue here.

Thank you for being so understanding!

**Reply by amd64fox** [reply]
When [creating a issue](https://github.com/amd64fox/SpotX/issues/new?assignees=&labels=%E2%9D%8C+bug&template=bug_report.yml), it is indicated what you should do before you submit it, read it more carefully

**Reply by ghost** [reply]
Ohh, I now see that I did not insert any computer information. Sincerest apologies!

**Reply by amd64fox** [reply]
not only information about the computer, you should study the FAQ, especially the first paragraph, I can tell you right away that your problem is not related to SpotX, look for a solution on Google or on the official Spotify forum, there is enough information about this, the main thing is to want to solve it and find it .
